packages/clerk-js/src/v4/elements/FlowCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { Card, descriptors, Flex } from '../customizables';
+import { generateFlowMetadataClassname } from '../customizables/classGeneration';
+import { CardStateProvider, FlowMetadata, FlowMetadataProvider } from './contexts';
+
+type FlowCardRootProps = React.PropsWithChildren<FlowMetadata>;
+
+const FlowCardRoot = (props: FlowCardRootProps): JSX.Element => {
+  const { children, flow, page, ...rest } = props;
+
+  return (
+    <FlowMetadataProvider
```

Comment:
❓ Can you please elaborate on this element? Does it block the UI from changing until a condition is met? It seems to me that we need a Wizard element.

Reply:
(this component has been changed and now also renders the InternalThemeProvider)

This one provides the necessary flow/page metadata, aka which flow/page the nested customisables should use as their relative position inside the `appearance` object.

It also renders an `InternalThemeProvider` that's responsible to provide the base `theme` for our styled system (used by emotion). This also needs to be handled on a per-flow basis, as the appearance object also supports nesting `variables` objects